3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -30,7 +30,7 @@ internal partial class BabylonExporter : ILoggingProvider
         public static string exporterVersion = "Custom.Build.Version";
         public float scaleFactorToMeters = 1.0f;
 
-        public const int MaxSceneTicksPerSecond = 4800; //https://knowledge.autodesk.com/search-result/caas/CloudHelp/cloudhelp/2016/ENU/MAXScript-Help/files/GUID-141213A1-B5A8-457B-8838-E602022C8798-htm.html
```

Comment:
I think for the sake of explaining our magic numbers, its a good idea to always include links to autodesk documentation where appropriate.

Reply:
Yes, however this link is broken..